charts/shopware/values.yaml

```diff
@@ -257,9 +259,59 @@ redissession:
   auth:
     enabled: false
 
-redisapp:
+valkeysession:
   enabled: true
   architecture: standalone
+  replica:
+    replicaCount: 1
+  master:
+    resources:
+      requests:
+        memory: 500Mi
+        cpu: 1000m
+      limits:
+        memory: 1Gi
+        cpu: 1000m
+  auth:
+    enabled: false
+
+valkeyworker:
+  enabled: false
+  architecture: standalone
+  replica:
+    replicaCount: 1
+  master:
+    resources:
+      requests:
+        memory: 500Mi
+        cpu: 1000m
+      limits:
+        memory: 1Gi
+        cpu: 2000m
+  auth:
+    enabled: false
+
+redisapp:
+  enabled: false
+  architecture: standalone
```

Comment:
If we don't enable it, we can remove all other values. If somebody needs it and use it for now, they will have values for it.